rocky/assets/js/renderNormalizerOutputOOIs.js

```diff
@@ -0,0 +1,45 @@
+
+const buttons = document.querySelectorAll(".expando-button.normalizer-list-table-row");
+
+buttons.forEach((button) => {
+  const raw_task_id = button.closest('tr').getAttribute('data-task-id');
+  const task_id = button.closest('tr').getAttribute('data-task-id').replace(/-/g, "");
+  const json_url = window.location.href + "/" + task_id;
+
+  const getJson = (url, callback) => {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
+    xhr.responseType = 'json';
+    xhr.onload = function() {
+      var status = xhr.status;
+      if (status === 200) {
+        callback(null, xhr.response);
+      } else {
+        callback(status, xhr.response);
+      }
+    };
+    xhr.send();
+  };
+
+
+  button.addEventListener("click", () => getJson(json_url, function(err, data) {
+    if (err !== null) {
+      alert('Somthing went wrong: ' + err);
```

Comment:
This could just be moved to the getJson function, no need to pass along the err variable, and then act on it here.
Also, maybe do a console.log of the error, and some visual feedback for the user telling him the 'fold open' did not go as planned?

Reply:
Yes, this definitely needs another fallback then to just show an alert.